getting-started-guides/notebook/python-notebook.md

```diff
@@ -8,21 +8,31 @@ It is assumed that the `SPARK_MASTER` and `SPARK_HOME` environment variables are
 
 1. Make sure you have [Jupyter notebook installed](https://jupyter.org/install.html). If you install it with conda, please makes sure your Python version is consistent.
 
-2. Make sure you have `SPARK_JARS` and `SPARK_PY_FILES` set properly. Please note, here *cudf-0.17-cuda10-1.jar* is used as an example. Please choose other *cudf-0.17* jars based on your environment. You may need to update these env variables because the working directory will be changed:
-  ```
-  export LIBS_PATH=[full path to xgboost4j_spark/libs]
-  export SPARK_JARS=${LIBS_PATH}/cudf-0.17-cuda10-1.jar,${LIBS_PATH}/xgboost4j_3.0-1.3.0-0.1.0.jar,${LIBS_PATH}/xgboost4j-spark_3.0-1.3.0-0.1.0.jar,${LIBS_PATH}/rapids-4-spark_2.12-0.3.0.jar
-  export SPARK_PY_FILES=${LIBS_PATH}/xgboost4j-spark_3.0-1.3.0-0.1.0.jar,${LIBS_PATH}/samples.zip
-  ```
+2. Make sure you have below jars.
+
+``` bash
+export CUDF_JAR=cudf-0.18-cuda10.1.jar
```

Comment:
Do we still need $LIBS_PATH?
Otherwise, the step 3 "Go to the project root directory and launch the notebook" can't find the files, I think.